client/src/components/NotificationPopUp/NotificationPopUp.tsx

```diff
@@ -60,8 +51,8 @@ const NotificationPopUp = (): JSX.Element => {
     filterNotification.length &&
       filterNotification?.map(async (notification) => await updateNotification(notification));
     setAnchorEl(event.currentTarget);
-    num === 0 ? setNum(1) : setNum(0);
-    setId(num);
+    //num === 0 ? setNum(1) : setNum(0);
```

Comment:
Please remove?

Reply:
sorry about that; i commented to remove it later but forgot. Done with the commit.